Doesn't work with react-native >=0.52

The output is:

   > $ whack run start
   > [whackage] init /Users/.../client-blocks -> node_modules/client-blocks
   > 
   > &gt; react-native start "--config" "/usr/local/lib/node_modules/whackage/src/packager/rn-cli.config.js"
   >
   > Scanning folders for symlinks in /Users/.../app/node_modules (19ms)
   >
   > Cannot find module '/Users/.../app/node_modules/react-native/packager/blacklist'

It looks like they renamed metro-bundler to metro [https://github.com/facebook/react-native/commit/654fed46f49b5002096ff55c2e8523af48b22c24](https://github.com/facebook/react-native/commit/654fed46f49b5002096ff55c2e8523af48b22c24)

--- a/src/packager/rn-cli.config.js
+++ b/src/packager/rn-cli.config.js
@@ -7,14 +7,19 @@ const log = require('../util/log');
 function getBlacklist() {
   var blacklist;
 
-  // RN >= 0.47
+  // RN >= 0.52
   try {
-    blacklist = require(`${process.cwd()}/node_modules/metro-bundler/src/blacklist`);
+    blacklist = require(`${process.cwd()}/node_modules/metro/src/blacklist`);
   } catch (e1) {
+    // RN >= 0.47
     try {
-      blacklist = require(`${process.cwd()}/node_modules/metro-bundler/build/blacklist`);
+      blacklist = require(`${process.cwd()}/node_modules/metro-bundler/src/blacklist`);
     } catch (e2) {
-      blacklist = require(`${process.cwd()}/node_modules/react-native/packager/blacklist`);
+      try {
+        blacklist = require(`${process.cwd()}/node_modules/metro-bundler/build/blacklist`);
+      } catch (e3) {
+        blacklist = require(`${process.cwd()}/node_modules/react-native/packager/blacklist`);
+      }
     }
   }
 

